### PR TITLE
Case 21079: move missed QtWebEngine references to +webengine directory 

### PR DESCRIPTION
**Pull request by danteruiz**
ticket - https://highfidelity.manuscript.com/f/cases/21079/Move-missed-QtWebEngine-references-to-webengine-directory

https://highfidelity.manuscript.com/f/cases/20879/Migrate-WebEngine-referencing-QML-into-a-webengine-folder-replace-top-level-item-with-stubs

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17141/HighFidelity-Beta-PR14877-d71fb2a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17141/HighFidelity-Beta-Interface-PR14877-d71fb2a.dmg

**Comment by JeffClinton** [comment]
Build this please

File: interface/resources/qml/QmlWebWindow.qml [hunk 1]
```diff
@@ -62,47 +60,6 @@ Windows.ScrollingWindow {
             url: "about:blank"
             anchors.fill: parent
             focus: true
```
Comment (jherico) [inline interface/resources/qml/QmlWebWindow.qml]: You've removed the `userScriptUrl`, but this field is referenced by the alias on line 26.  Either change the alias to a string in this version, or retain the property here (even though it's unused).

File: interface/resources/qml/controlsUit/WebSpinner.qml [hunk 2]
```diff
@@ -9,10 +9,14 @@
 //
 
 import QtQuick 2.5
-import QtWebEngine 1.5
 
 AnimatedImage {
-    property WebEngineView webview: parent
+    Item {
```
Comment (jherico) [inline interface/resources/qml/controlsUit/WebSpinner.qml]: I would suggest replacing this `AnimatedImage` with simply an `Image` pointing to something that indicates web content can't be displayed... like this: 
image

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17144/HighFidelity-Beta-PR14877-d71fb2a.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17144/HighFidelity-Beta-PR14877-d71fb2a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17144/HighFidelity-Beta-Interface-PR14877-d71fb2a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17144/HighFidelity-Beta-PR14877-d71fb2a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17144/HighFidelity-Beta-Interface-PR14877-d71fb2a.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17158/HighFidelity-Beta-PR14877-76a56c6.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17158/HighFidelity-Beta-PR14877-76a56c6.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17158/HighFidelity-Beta-Interface-PR14877-76a56c6.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17158/HighFidelity-Beta-PR14877-76a56c6.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17158/HighFidelity-Beta-Interface-PR14877-76a56c6.exe

**Pull request merged**